lib/main-window.js

```diff
@@ -39,6 +39,7 @@ const windowSize = config.get('windowsize') || {
   width: 1080,
   height: 720
 }
+const zoomFactor = config.get('zoomFactor') || 1.0
```

Comment:
I don't remember the value `zoomFactor` being saved to config, can you explain where this `zoomFactor` come from? I checked `ZoomManager` but it saving the value directly to the user config file with the key `zoom` not `zoomFactor`

Reply:
This `zoomFactor` come from [the electron docs about zoom settings](https://electronjs.org/docs/api/web-frame).
This docs uses `zoomFactor` or `zoomLevel` which are the zoom settings value.
`ZoomManager` uses `zoomFactor`, so I used `zoomfactor`.

But I checked `ZoomManager` again.
The zoom settings value saves to `zoom` in `ConfigManager`.
Should I change from `config.get('zoomFactor')` to `config.get('zoom')`?

Reply:
@roottool I'm really sorry for the late response. The thing is the zoom factor is saved using the ConfigManager which save to the local storage instead of the config file using electron config so your code here won't be able to retrieve the zoom factor. You will have to communicate with the rendering process to get the factor from the localStorage.
https://github.com/BoostIO/Boostnote/blob/1dd71fc923a5b972bf6685ca4b7aab41faf0f3d5/browser/main/lib/ConfigManager.js#L126-L128